From 39eff609f917c7f301572b8076a001cb9621eea2 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 16 Feb 2024 09:42:02 +0100
Subject: [PATCH 0687/1695] dts: stm32: enable PKA on stm32mp257f-dk

Set PKA status to "okay" in stm32mp257f-dk.dts

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 73b9121333ff93d604605df5a2d5ddb12e00114f dts: stm32: enable PKA on stm32mp257f-dk]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I5bfe3c6decbb8424efeda68fe04d68fc5c619749
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/360597
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 35b529ddc..0cfc192cc 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -319,6 +319,10 @@
 	status = "okay";
 };
 
+&pka {
+	status = "okay";
+};
+
 &pwr {
 	status = "okay";
 	wakeup-gpios = <&gpioa 0 GPIO_ACTIVE_HIGH>, <0>, <0>, <0>, <0>, <0>;
-- 
2.47.2

